Makefile

```diff
@@ -8,13 +8,31 @@ up:
 
 migrate:
 	docker-compose run --rm web python manage.py migrate
+	docker-compose down
+
+migzero:
+	docker-compose run --rm web python manage.py migrate api zero
+	docker-compose down
 
 makemigrations:
 	docker-compose run --rm web python manage.py makemigrations
+	docker-compose down
 
 down:
 	docker-compose down
 
-#utility: removes all images and containers related to terra-mars
+#utility: removes all images and containers related to terra-mars-api and <none>
 clear:
 	 docker-compose down && docker images -a | egrep "<none>|terra-mars-api*" | awk '{print $3}' | xargs docker rmi
+
+collectstatic:
+	docker-compose run web python manage.py collectstatic
+	docker-compose down
```

Comment:
quite a lot of duplication - you can define the command as a variable at the top